docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/monolithic-applications.md

```diff
@@ -27,42 +27,52 @@ The monolithic approach is common, and many organizations are developing with th
 
 From an infrastructure perspective, each server can run many applications within the same host and have an acceptable ratio of efficiency in your resources usage, as shown in Figure 4-2.
 
-![](./media/image2.png)
+![A single host can run multiple apps in separate containers.](./media/image2.png)
 
-Figure 4-2: A host running multiple apps/containers
+**Figure 4-2.** A host running multiple apps/containers
 
-You can deploy monolithic applications in Azure by using dedicated VMs for each instance. Using [Azure VM Scale Sets](https://docs.microsoft.com/azure/virtual-machine-scale-sets/), you can scale the VMs easily. [Azure App Services](https://azure.microsoft.com/services/app-service/) can run monolithic applications and easily scale instances without having to manage the VMs. Since 2016, Azure App Services can run single instances of Docker containers, as well, simplifying the deployment. And, using Docker, you can deploy a single VM as a Docker host and run multiple instances. Using the Azure balancer, as illustrated in the Figure 4-3, you can manage scaling.
+Finally, from availability perspective, monolithic applications must be deployed as a whole, that means that in case you must *stop and start*, all functionality and users will be affected during the deployment window. In certain situations, the use of Azure and containers can minimize these situations and reduce the probability of downtime of your application, as you can depict viewing Figure 4-3.
```

Comment:
```suggestion
Finally, from an availability perspective, monolithic applications must be deployed as a whole; that means that in case you must *stop and start*, all functionality and all users will be affected during the deployment window. In certain situations, the use of Azure and containers can minimize these situations and reduce the probability of downtime of your application, as you can see in Figure 4-3.
```